clang/test/SemaCXX/warn-unsafe-buffer-usage-libc-functions.cpp

```diff
@@ -155,3 +158,47 @@ void ff(char * p, char * q, std::span<char> s, std::span<char> s2) {
   wcscpy_s();
 #pragma clang diagnostic pop
 }
+
+#else
+
+#include <ptrcheck.h>
+typedef unsigned size_t;
+
```

Comment:
Some of these new tests look unrelated to this change.  Please consider splitting them out to a separate PR.

Reply:
sure.  These are things left un-upstreamed.